en/lessons/specifics/benchmarking.md

```diff
@@ -0,0 +1,346 @@
+---
+version: 1.0.0
+title: Benchmarking 
+redirect_from:
+  - /lessons/specifics/benchmarking/
+---
+
+We can't just guess about which functions are fast and which are slow - we need actual measurements when we're curious. That's where benchmarking comes in. In this lesson, we'll learn about how easy it is to measure the speed of our code.
+
+{% include toc.html %}
+
+# About Benchee 
+
+While there is a [function in Erlang](http://erlang.org/doc/man/timer.html#tc-1) that can be used for very basic measurement of a function's execution time, it's not as nice to use as some of the available tools, so we're going to use [Benchee](https://github.com/PragTob/benchee). Benchee provides us with a range of statistics with easy comparisons between scenarios, a great feature that allows us to test different inputs to the functions we're benchmarking, and several different formatters that we can use to display our results, as well as the ability to write your own formatter if desired.
```

Comment:
I might be too eager about the topic but I think it's worth mentioning that tc-1 only does one measurement and ideally you want to make many measurements to get a better view of the overall distribution + avoid just having your one measurement be an outlier, some warmup time and such.